htdocs/web_portal/views/service_group/view_sgroup.php

```diff
@@ -7,30 +7,55 @@
 
 <script type="text/javascript" src="<?php echo \GocContextPath::getPath()?>javascript/confirm.js"></script>
 <div class="rightPageContainer">
-    <div style="float: left;">
-        <img src="<?php echo \GocContextPath::getPath()?>img/virtualSite.png" class="pageLogo" />
-    </div>
-    <div style="float: left;">
-        <h1 style="float: left; margin-left: 0em;">
-                Service Group: <?php xecho($params['sGroup']->getName())?>
-        </h1>
-        <span style="clear: both; float: left; padding-bottom: 0.4em;"><?php xecho($params['sGroup']->getDescription())?></span>
-    </div>
+    <div
+        style="display: flex;
+            justify-content: space-between;
+            gap:10%"
+    >
+            <div style="display:flex; flex-direction: row;">
```

Comment:
I think this is slightly over indented compared to the above line?

Reply:
Ah right. Updated the code in eae6644c0ac151f6a1840ac81e274a0c22e99735